File: app/models/answer.rb
# == Schema Information
#
# Table name: answers
#
#  id          :bigint(8)        not null, primary key
#  correct     :boolean          default(FALSE), not null
#  description :text             not null
#  created_at  :datetime         not null
#  updated_at  :datetime         not null
#  question_id :integer          not null
#

class Answer < ApplicationRecord
  belongs_to :question
  has_many :referee_answers, dependent: :destroy
end

File: dotnet/ManagementHub.Models/Answer.cs
using System;
using System.Collections.Generic;

namespace ManagementHub.Models
{
    public partial class Answer
    {
        public Answer()
        {
            RefereeAnswers = new HashSet<RefereeAnswer>();
        }

        public long Id { get; set; }
        public long QuestionId { get; set; }
        public string Description { get; set; } = null!;
        public bool Correct { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }

        public virtual Question Question { get; set; } = null!;
        public virtual ICollection<RefereeAnswer> RefereeAnswers { get; set; }
    }
}
